### DISCUSSION TITLE
Send E-Mails when builds are failing only

### DISCUSSION THREAD

**Discussion by gliden** [codemagic-ci-cd/codemagic-docs#2270] [Ideas]
### Use case

```Text
We're building our apps twice a day and publish it to AppCenter. 
Normally we don't need the notification mail, but when the build fails the notification is necessary.
```

### Idea

```Text
Add a option to the E-Mail notification setting.
```

**Comment by Ycoi** [comment]
Hi! This is already possible when using the yaml configuration. See the [docs](https://docs.codemagic.io/yaml-notification/email/) for more details. Yaml configuration gives you also otherwise more flexibility, so if you you just are comfortable on using yaml, I can recommend switching.

We do not have immediate plans to add this to the workflow editor but I’ll discuss this with the team. In the long run we are looking into moving to the direction of a visual yaml editor and making using yaml easier instead of the current workflow editor. This is because yaml allows more flexibility and we want to give more of our users access to that flexibility.

**Comment by lukemmtt** [comment]
+1 for this—I use the workflow editor at the moment, and I would really love to get notifs for build failures only.
